### PR TITLE
Update documentation for version 2.1.0

### PR DESCRIPTION
**Pull request by GuillaumeTh**
Update documentation for version 2.1.0

### CODE REVIEW

File: source/pipeline/launch.rst [hunk 1]
```diff
@@ -83,10 +57,37 @@ must be saved in ``.sh`` file (e.g. ``cmd.sh``) to be executed with ``sbatch``.
...
+    singularity.runOptions="--bind PATH_TO_DATA"
+
+Where ``PATH_TO_DATA`` is the path to your storage disk.
+
+Then run the following command:
+
+::
+
+    # With Singularity
+    $> nextflow -c singularity.conf run tractoflow/main.nf --root input_folder --dti_shells "DTI_SHELLS" --fodf_shells "FODF_SHELLS" -with-singularity singularity_name.img -resume
```
Comment (arnaudbore) [inline source/pipeline/launch.rst]: Maybe add the bids version also.

File: source/pipeline/profiles.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    $> nextflow run tractoflow/main.nf --root input_folder --dti_shells "DTI_SHELLS" --fodf_shells "FODF_SHELLS" -profile macos,fully_reproducible -with-singularity singularity_name.img -resume
+
+Profiles available
+------------------
+
+``macos``
+    When this profile is used, TractoFlow will modify a parameter (scratch) for MacOS users.
+
+``use_cuda``
```
Comment (arnaudbore) [inline source/pipeline/profiles.rst:18]: default ?

File: source/pipeline/profiles.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    When this profile is used, TractoFlow will use eddy_cuda for Eddy process. This feature is available with NVidia GPUs only.
+
+``fully_reproducible``
```
Comment (arnaudbore) [inline source/pipeline/profiles.rst:22]: default ?

**Review by arnaudbore — COMMENTED** [review]
Yeah !! Tractoflow 2.1.0 !!

File: source/pipeline/profiles.rst (hunk 3 above)
Comment (GuillaumeTh) [reply source/pipeline/profiles.rst:22]: I don't think so because longer.

File: source/pipeline/profiles.rst (hunk 2 above)
Comment (GuillaumeTh) [reply source/pipeline/profiles.rst:18]: I don't think so. You must to have an NVIDIA GPU and for example it is not the case on the clusters

File: source/pipeline/launch.rst (hunk 1 above)
Comment (GuillaumeTh) [reply source/pipeline/launch.rst]: I will add this version

File: source/pipeline/profiles.rst (hunk 2 above)
Comment (arnaudbore) [reply source/pipeline/profiles.rst:18]: Can you explain what is the default behavior ?

File: source/pipeline/profiles.rst (hunk 3 above)
Comment (arnaudbore) [reply source/pipeline/profiles.rst:22]: Can you explain what is the default behavior ?

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**